### ISSUE TITLE
[Info] How use proxies?

### ISSUE THREAD

**Issue by xion87** [games647/ChangeSkin#149]
I got 2 vps (teamspeak and sites) and the dedicated where i run minecraft.
There is a guide for setup these proxy? How i do it? 
Thank you

**Comment by xion87** [comment]
that was a error on my side, i fixed it by whitelisting, but my server is unable to reach to squid (no error)

**Comment by xion87** [comment]
@games647 any clue why is not working? I use this config https://gist.github.com/xion87/0b4d6cdae5b57ae504b9a05fa63756c5
There is no squid log about connections

**Comment by games647** [comment]
Do you created an TLS certificate and installed it on your server?

**Comment by xion87** [comment]
tls certificate on the vps where squid is installed?

**Comment by games647** [comment]
Yes on that server and the configuration that your client (the Spigot server), actually trusts this certificate. For example by using a Let's Encrypt one.

**Comment by xion87** [comment]
changeskin is installed on bungeecord, so what i have to do? maybe if you can join discord and help i would be grateful :), you have me on friend list and i contacted you
